ui/admin/app/components/auth-methods/auth-method/managed-groups/managed-group/actions/index.hbs

```diff
@@ -4,17 +4,12 @@
 }}
 
 {{#if (can 'delete model' @model)}}
-  <Rose::Dropdown
-    @text={{t 'actions.manage'}}
-    @dropdownRight={{true}}
-    as |dropdown|
-  >
-    <dropdown.button
-      @style='danger'
-      @disabled={{@model.canSave}}
+  <Hds::Dropdown data-test-manage-managedgroups as |dd|>
```

Comment:
```suggestion
  <Hds::Dropdown data-test-manage-managed-group-dropdown as |dd|>
```
I think we have been adding `-dropdown` to these test selectors.

Reply:
I wondered why that looked so off lol